### PR TITLE
[DOCS] Added release highlights for 6.3

### PR DESCRIPTION
**Pull request by debadair**
Pulled the items from the draft blog post.

### CODE REVIEW

File: docs/reference/release-notes/highlights-6.3.0.asciidoc [hunk 1]
```diff
@@ -4,6 +4,45 @@
...
+storage space. This is particularly useful when you're working with monitoring
+and metrics data where it's not feasible to retain the raw data indefinitely.
+
+When you ask Elasticsearch to store a rollup of data, it also stores the underlying
+statistics. For example, if you roll up an average, the sum and count are also
+stored so that the average can be recomputed at query time. This enables you to
+query both rolled up data and “live” data simultaneously using the standard
+query DSL.
+
+For more information, see <<xpack-sql, SQL Access>>.
```
Comment (pcsanwald) [inline docs/reference/release-notes/highlights-6.3.0.asciidoc]: shouldn't this be a link to rollups instead of SQL?

**Review by pcsanwald — APPROVED** [review]
one small thing I think is just copy-paste mistake?

File: docs/reference/release-notes/highlights-6.3.0.asciidoc (hunk 1 above)
Comment (debadair) [reply docs/reference/release-notes/highlights-6.3.0.asciidoc]: Yup. Darned copy/paste. :-)

File: docs/reference/release-notes/highlights-6.3.0.asciidoc [hunk 2]
```diff
@@ -4,6 +4,45 @@
...
+of changes in this release.
+
+NOTE: We want your feedback on the experimental features in this release! Let
+us know what you’d like to see next and any problems you encounter.
+
+=== SQL
+This experimental feature enables users who are familiar with SQL to
+use SQL statements to query Elasticsearch indices. In addition to querying
+through the SQL API, you can use the Translate API to see how SQL queries
+are translated to native ELasticsearch queries.
```
Comment (lcawl) [inline docs/reference/release-notes/highlights-6.3.0.asciidoc]: Small typo: "ELas..."

**Pull request merged**